app/model/editions/templates/TheDummyEdition.scala

```diff
@@ -7,7 +7,7 @@ import model.editions._
 import model.editions.templates.TemplateHelpers._
 
 //noinspection TypeAnnotation
-object TheDummyEdition extends EditionDefinitionWithTemplate {
+object TheDummyEdition extends RegionalEdition {
```

Comment:
NO!

If you're going to do it this way, this has to be something else.

Reply:
maybe I just called it `BaseEdition` rather than `RegionalEdition`?